src/pudl/transform/gridpathratk.py

```diff
@@ -0,0 +1,57 @@
+"""Transformations of the GridPath RA Toolkit renewable generation profiles."""
+
+import pandas as pd
+from dagster import asset
+
+
+def _transform_profiles(
+    profiles: pd.DataFrame, utc_offset: pd.Timedelta
+) -> pd.DataFrame:
+    """Basic transformations that can be applied to many profiles.
+
+    - Construct a datetime column and adjust it to be in UTC.
+    - Reshape the table from wide to tidy format.
+    - Name columns appropriately.
+    """
+    # Convert timestamp to UTC before stacking!
+    # TODO: Ensure that we're using the same hour starting vs. ending convention
+    # here as we use in the FERC-714 and EPA CEMS data. Should this be reflected in
+    # the column names here and elsewhere, whatever the convention is?
+    datetime_cols = ["year", "month", "day", "hour"]
+    profiles["utc_datetime"] = (
+        pd.DatetimeIndex(pd.to_datetime(profiles.loc[:, datetime_cols])) - utc_offset
+    )
+    profiles = (
+        profiles.drop(columns=datetime_cols)
+        .set_index("utc_datetime")
+        .stack()
+        .reset_index()
+    )
+    profiles.columns = ["utc_datetime", "aggregation_key", "capacity_factor"]
```

Comment:
are you ensuring these columns are in this order? this seems more brittle than a rename.

Reply:
Unfortunately the columns have no names until this assignment happens.